### PR TITLE
Permissions with CLI export templates

### PR DESCRIPTION
**Pull request by jstirnaman**
Part of #3267

Stress and clarify the role of permissions when exporting templates. Removed use of --org-name due to some confusing behavior (influx-cli issue https://github.com/influxdata/influx-cli/issues/309  referenced in #3267).
Trying to strike a balance b/w documenting the way it is vs. the way it should be.
Haven't yet tested the API without CLI.

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.1/influxdb-templates/create.md [hunk 1]
```diff
@@ -16,64 +16,70 @@ related:
...
 create InfluxDB templates.
-Add resources (buckets, Telegraf configurations, tasks, and more) in the InfluxDB
+Add [resources](/influxdb/v2.1/influxdb-templates/#template-resources) (buckets, Telegraf configurations, tasks, and more) in the InfluxDB
 UI and export the resources as a template.
 
 {{< youtube 714uHkxKM6U >}}
 
 {{% note %}}
-Templatable resources are scoped to a single organization, so the simplest way to create a
+[Templatable resources](/influxdb/v2.1/influxdb-templates/#template-resources) are scoped to a single organization, so the simplest way to create a
```
Comment (jstirnaman) [inline content/influxdb/v2.1/influxdb-templates/create.md:25]:
```suggestion
[Template resources](/influxdb/v2.1/influxdb-templates/#template-resources) are scoped to a single organization, so the simplest way to create a
```

File: content/influxdb/v2.1/influxdb-templates/create.md [hunk 2]
```diff
@@ -16,64 +16,70 @@ related:
...
+Do one of the following to export resources to template manifests:
 
 - [Export all resources in an organization](#export-all-resources)
 - [Export specific resources in an organization](#export-specific-resources)
 - [Export a stack and its associated resources](#export-a-stack)
 
 ### Export all resources
 
-To export all templatable resources within an organization to a template manifest,
+To export all [templatable resources](/influxdb/v2.1/influxdb-templates/#template-resources) within an organization to a template manifest,
```
[the remark this answers is not in the document]
Comment (jstirnaman) [reply content/influxdb/v2.1/influxdb-templates/create.md:47]: @kelseiv This seems to be an important distinction: exporting everything that's allowed vs. selectively exporting. Why remove it?

[the remark this answers is not in the document]
Comment (kelseiv) [reply content/influxdb/v2.1/influxdb-templates/create.md:47]: @jstirnaman, agree, if introducing the "Export all resources" section, let's keep "all" in (and drop the rest) :-)

Alternatively, if you find it streamlines the reader experience, we could do "Export resources" section & bump down "Export all resources" and "Export specific resources"

File: content/influxdb/v2.1/influxdb-templates/_index.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ influxdb/v2.1/tags: [templates]
 
 InfluxDB templates are preconfigured InfluxDB resources and can contain everything
```
Comment (pierwill) [inline content/influxdb/v2.1/influxdb-templates/_index.md:11]:
```suggestion
InfluxDB templates are preconfigured InfluxDB resources that can contain everything
```

**Comment by jstirnaman** [comment]
Setting this hotness back to *draft* for now.

**Review by kelseiv — APPROVED** [review]

**Pull request merged**